main.go

```diff
@@ -214,8 +213,14 @@ type PackageInfo struct {
 	License string `json:"license"`
 }
 
+//go:embed CREDITS.npm.json
+var creditsNpmJson []byte
+
+//go:embed CREDITS.go.json
+var creditsGoJson []byte
+
 func serveCredits(w http.ResponseWriter, req *http.Request) {
-	json := mustAsset("/CREDITS.npm.json")
+	json := creditsNpmJson
```

Comment:
Is it intended behavior `/credits` shows only npm dependencies?

Reply:
Yes, because `/credits` is to show licenses of files distributed through browser accesses.  No go binary is distributed in this case.